samples/Azure.Analytics.Purview.Account/Generated/CollectionsClient.cs

```diff
@@ -0,0 +1,771 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Analytics.Purview.Account
+{
+    /// <summary> The Collections service client. </summary>
+    public partial class CollectionsClient
```

Comment:
Per the [subclient guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-subclients) subclients shouldn't have the `Client` suffix.  And this makes me think that `Collections` is going to be too general a name as well.  Have we given any thought to automating renaming in these instances?  IIRC, I saw a proposal go by that suggesting prefixing a resource name with the service name - @ellismg, do you know if we closed on this?

Reply:
@annelo-msft I don't think we made a decision.  We may just want to start with "use the operation group name as the type name (without the `Client` suffix, unless the sub client is also supposed to be publicly constructible?)" and allow someone to use the normal strategy to rename the generated type if we want to add a prefix.

I do think that the prefix should be on the type name only, not also in the name of the factory method that mints a sub client (i.e. `PurviewCollections` as the type name but still `GetCollectionsClient()` instead of `GetPurviewCollectionsClient()`)

Reply:
@AlexanderSher, has this been decided in your subclients PR?